src/Login/ProtectedContent.jsx

```diff
@@ -215,6 +219,37 @@ class ProtectedContent extends React.Component {
     return newState;
   };
 
+  /**
+   * Display access contol message if userAccessToSite property set and user does not have sufficient privileges
+   */
+  checkSiteAccess = (initialState) => {
+    // check for setup property
+    if (userAccessToSite?.enabled && initialState.user.resources) {
+      const checkIfUserHasPermission = (resources) => {
```

Comment:
the name of this function is a bit confusing, so if it returns true it means user doesn't have premission and should be redirected, and false means user has premission
This is conter-intuitive, suggest to get it a better name

Reply:
changed the function name to userHasNoPermissions, need to redo the whole section to make it more readable